gcc-4.8/post-link.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+if [ "$(uname)" != "Darwin" ]; then
```

Comment:
@asmeurer  This file is not only for OS X though. However, since you said that post-list.sh shouldn't be included on OS X. Could you please say why?